image.go

```diff
@@ -337,6 +338,9 @@ func WithUnpackDuplicationSuppressor(suppressor kmutex.KeyedLocker) UnpackOpt {
 }
 
 func (i *image) Unpack(ctx context.Context, snapshotterName string, opts ...UnpackOpt) error {
```

Comment:
Is this a different workflow for image unpacking? we should already have tracing around the [unpacker](https://github.com/containerd/containerd/blob/main/pkg/unpack/unpacker.go#L161) invoked during `Pull`

Reply:
Yes this is different from the unpack call path